table/block_based/filter_policy_internal.h

```diff
@@ -38,6 +39,31 @@ class BuiltinFilterBitsBuilder : public FilterBitsBuilder {
   virtual double EstimatedFpRate(size_t num_entries, size_t bytes) = 0;
 };
 
+// Used for testing, and as a base class for SkipFilterBitsBuilder
+class AlwaysTrueBitsBuilder : public BuiltinFilterBitsBuilder {
+ public:
+  size_t CalculateSpace(size_t) override { return 6U; }
+  double EstimatedFpRate(size_t, size_t) override { return 1.0; }
+  size_t ApproximateNumEntries(size_t) override { return SIZE_MAX; }
+  void AddKey(const Slice&) override {}
+  Slice Finish(std::unique_ptr<const char[]>* buf) override {
+    // Interpreted as "always true" filter (0 probes over 1 byte of
+    // payload, 5 bytes metadata)
+    buf->reset(new char[6]{});
+    return Slice(buf->get(), 6);
+  }
+};
+
+// See FilterBitsBuilder::ShouldSkipFilter
+class SkipFilterBitsBuilder : public AlwaysTrueBitsBuilder {
```

Comment:
Should `SkipFilterBitsBuilder` based on `AlwaysTrueBitsBuilder` or it could inherit from `BuiltinFilterBitsBuilder` directly?